api/helper/process-logger-results.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/**
+ * @module api/helper/process-logger-results
+ */
+
+/**
+ * Takes in a survey_instance model and processes it into a format recognized by app.
+ * @param {Object} logs - list of objects of survey result logs
+ * @returns {Object} processed list of logs in an array
+ */
+function processLoggerResults (logs) {
+    return {
+        eventName: logs.eventName,
+        metaData: JSON.stringify(logs.metaData),
```

Comment:
As of now we cannot determine what the size limit will be. The app will log all button interactions any number of times a user clicks. It is difficult to judge what would be the interaction pattern at this point.